Source/WebCore/Modules/webaudio/AudioParam.cpp

```diff
@@ -328,17 +330,17 @@ void AudioParam::calculateFinalValues(float* values, unsigned numberOfValues, bo
 
     // If we're not sample accurate, duplicate the first element of |values| to all of the elements.
     if (!sampleAccurate)
-        std::fill_n(values + 1, numberOfValues - 1, values[0]);
+        std::ranges::fill(values.subspan(1), values[0]);
 
     // As per https://webaudio.github.io/web-audio-api/#computation-of-value, we should replace NaN values
     // with the default value.
-    replaceNaNValues(values, numberOfValues, m_defaultValue);
+    replaceNaNValues(values, m_defaultValue);
 
     // Clamp values based on range allowed by AudioParam's min and max values.
-    VectorMath::clamp(values, minValue(), maxValue(), values, numberOfValues);
+    VectorMath::clamp(values.data(), minValue(), maxValue(), values.data(), values.size());
```

Comment:
Seems like this needs to change to take a span. I'm sure you noticed.

Reply:
Yes, I've decided to keep VectorMath out of this PR to keep the patch size reasonable.